Quaver.Shared/Screens/Select/SelectScreen.cs

```diff
@@ -107,6 +107,7 @@ public override void Update(GameTime gameTime)
         {
             KeepPlayingAudioTrackAtPreview();
             HandleInput();
+            HandleMousePressRight();
```

Comment:
This should be located in `HandleInput()`, not here.

Reply:
Aight I'll change it real quick